utils/display/flink_jobs.go

```diff
@@ -0,0 +1,43 @@
+package display
+
+import (
+	"fmt"
+
+	"github.com/alexeyco/simpletable"
+
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
+)
+
+func FlinkJobsTable(flinkJobs []*meroxa.FlinkJob) string {
+	if len(flinkJobs) != 0 {
```

Comment:
You could early return here by checking if it's equal to 0 and the return.

REFERENCE: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow